docs/src/assets/logo-build.jl

```diff
@@ -0,0 +1,37 @@
+using Distributions, Plots, Colors
+
+# Gaussian with known variance
+# mu ~ Normal(mu0, sig0)
+# x_i ~ Normal(mu, sig_like)
+# Ref: https://en.wikipedia.org/wiki/Conjugate_prior#Continuous_distributions
+function compute_posterior_parameters(xs, mu0, sig0, sig_like)
+    n = length(xs)
+    var = inv(1 / sig0^2 + n / sig_like^2)
+    mu = var * (mu0 / sig0^2 + sum(xs) / sig_like^2)
+    return mu, sqrt(var)
+end
+
+mu0 = 0.0
+sig0 = 1.0
+sig_like = 1.0
+prior = Normal(mu0, sig0)
+# Inference
```

Comment:
Maybe add a space before or after the comment line. Depending on the code the comment refers to.

Reply:
Updated this in `logo-build.jl`.